### PR TITLE
fix!: LDAP sync triggers multiple cron jobs in case an invalid sync interval is provided

### PR DESCRIPTION
**Pull request by matheusbsilva137**
## Proposed changes (including videos or screenshots)

 - Validate LDAP sync intervals before setting up their corresponding cron jobs;
 - Use LDAP sync interval setting's `packageValue`s as the fallback value in case the provided value is invalid;
 - Add a migration to update LDAP sync interval setting's `packageValue`s (from cron description to actual cron expressions, which is the type of value expected by RC since 6.3).

## Steps to test or reproduce

Go to **Settings > LDAP > Premium** and set an invalid Avatar Background Sync Interval or Auto Logout Interval.
**Current behavior:** cron jobs are triggered successively (there is no interval between them).
**Expected behavior:** cron jobs intervals should follow each setting's default value instead.

## Further comments

[CORE-273](https://rocketchat.atlassian.net/browse/CORE-273)

[CORE-273]: https://rocketchat.atlassian.net/browse/CORE-273?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/i18n/package.json
Comment (MarcosSpessatto) [inline packages/i18n/package.json:1]: Let's discard this unrelated change

File: apps/meteor/server/startup/migrations/v305.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { Settings } from '@rocket.chat/models';
+import { isValidCron } from 'cron-validator';
+
+import { settings } from '../../../app/settings/server';
```
Comment (MarcosSpessatto) [inline apps/meteor/server/startup/migrations/v305.ts:4]: We should not use the cached setting in migrations, let's use the model instead.

**Review by MarcosSpessatto — CHANGES_REQUESTED** [review]

File: packages/i18n/package.json
Comment (matheusbsilva137) [reply packages/i18n/package.json:1]: Done! This is being changed on install, though
I'll check if this change is already on develop

**Review by MarcosSpessatto — DISMISSED** [review]
